Dockerfile

```diff
@@ -1217,11 +1217,12 @@ COPY --from=sdk-plus-lz4 /usr/share/licenses/lz4 /usr/share/licenses/lz4
 # a format that's convenient for extracting later.
 COPY --from=toolchain-archive /toolchain /toolchain
 
-# "sdk-libc-musl" has the musl C library and headers. We omit "sdk-libc-gnu"
-# because we expect to build glibc again for the target OS, while we will use
-# the musl artifacts directly to generate static binaries such as migrations.
+# "sdk-libc-musl" has the MUSL C library and headers.
 COPY --from=sdk-libc-musl / /
 
+# "sdk-libc-gnu" has the GNU C library and headers.
+COPY --from=sdk-libc-gnu / /
```

Comment:
> This relies a lot on glibc's backwards compatibility, and on making sure the SDK's version of glibc never gets ahead of the core kit's.

Is there going to be a check similar to the check done for `binutils` to prevent the `glibc` copy in the core kit from getting out of sync with the `glibc` version in the SDK?

Reply:
> Is there going to be a check similar to the check done for binutils to prevent the glibc copy in the core kit from getting out of sync with the glibc version in the SDK?

I expect it to be OK for the SDK copy of glibc to be older; it just can't be newer.